artiq/coredevice/comm_analyzer.py

```diff
@@ -762,12 +759,15 @@ def decoded_dump_to_target(manager, devices, dump, uniform_interval):
         manager.set_start_time(start_time)
     t0 = start_time
     for i, message in enumerate(messages):
-        if message.channel in channel_handlers:
+        if isinstance(message, StoppedMessage):
+            stopped_messages.append(message)
+            logger.debug(f"StoppedMessage at {get_message_time(message)}")
+        elif message.channel in channel_handlers:
             t = get_message_time(message)
             if t >= 0:
                 if uniform_interval:
                     interval.set_value_double((t - t0)*ref_period)
-                    manager.set_time(i)
+                    manager.set_time(i - len(stopped_messages))
```

Comment:
to make it uniform without holes in places of StoppedMessage

Reply:
Before this patch, we strip the `StoppedMessage` from the dump, this removes it as well. If we want to increment for the discarded `StoppedMessage` as well, this can be removed. I am not too sure about the use case for `uniform_interval`.

Reply:
understood, then probably this line can be removed.